### ISSUE TITLE
Create user configuration for attemptTimeout

### ISSUE THREAD

**Issue by vanreece** [Polymer/tachometer#228]
Right now the attemptTimeout is just set to 10s, but it could be more useful to have this configurable in the json configuration.

I tried to find existing issues about this and couldn't find anything, so I'm opening a new issue for it.

**Comment by aomarks** [comment]
Makes sense, thanks for filing. Do you have a benchmark that is expected to take more than 10 seconds?

**Comment by vanreece** [comment]
Yeah, I'm using this as an end to end performance test for a heavy web app, and it turns out that fully loading before starting the benchmark takes long enough that I'm right on that threshold of 10s. Maybe not the intended use case for tachometer, but it works nicely for it!
